### PR TITLE
[EWS V2] Handle ErrorMimeContentConversionFailed error

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [XSUP-28730](https://jira-dc.paloaltonetworks.com/browse/XSUP-28730)

## Description
Add handling for error object `ErrorMimeContentConversionFailed` that can be returned in the stream of results when fetching email objects.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py [hunk 1]
```diff
@@ -1020,6 +1019,14 @@ def fetch_last_emails(account, folder_name='Inbox', since_datetime=None, exclude
                 'Got an error when pulling incidents. You might be using the wrong exchange version.'
             ), exc)
             raise exc
+        except ErrorMimeContentConversionFailed as exc:
+            demisto.debug(f"Encountered an ErrorMimeContentConversionFailed error object while iterating: {exc}.\
+                Continuing to next item.")
+            continue
+        except AttributeError as exc:
+            demisto.debug(f"Encountered an AttributeError error object while iterating: {exc}.\
```
Comment (JasBeilin) [inline Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py:1027]:
```suggestion
            demisto.debug(f"Encountered an Attribute error object while iterating: {exc}.\
```

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2_test.py
Comment (JasBeilin) [inline Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2_test.py:1]: The test makes sure that the value error still errors the same?
Can we add a test to make sure that if we receive a list like [incident, error, incident] we return 2 incidents?

**Review by JasBeilin — CHANGES_REQUESTED** [review]
looks great, see my comments.

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2_test.py
Comment (samuelFain) [reply Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2_test.py:1]: Yes. the UT makes the differentiation between the different types of errors.
Also, I've added another UT to test the use-case you described in the second line of the comment.

**Review by JasBeilin — APPROVED** [review]

**Comment by samuelFain** [comment]
@yuvalbenshalom
Requesting force merge due to the following issue which fails the build: https://jira-dc.paloaltonetworks.com/browse/CIAC-8733

**Pull request merged**